### PR TITLE
utils_misc: APIs to get pci and interface info from host/guest

### PR DESCRIPTION
**Pull request by balamuruhans**
Based on the card name pci devices in host or hotplugged devices in
guest, PCI id and interfaces can be retrieved using these APIs

Signed-off-by: Balamuruhan S <bala24@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by balamuruhans** [comment]
```
2017-07-06 09:04:20,935 client           L1078 DEBUG| Sending command: lspci | awk '/MT27700 Family/ {print $1}'
2017-07-06 09:04:21,045 client           L1078 DEBUG| Sending command: echo $?
2017-07-06 09:04:21,146 sr_iov_hotplug   L0202 DEBUG| PCIs associated with MT27700 Family - ['00:01.0']
2017-07-06 09:04:21,146 client           L1078 DEBUG| Sending command: ifconfig -a
2017-07-06 09:04:21,252 client           L1078 DEBUG| Sending command: echo $?
2017-07-06 09:04:21,353 client           L1078 DEBUG| Sending command: ethtool -i eth0 | awk '/bus-info/ {print $2}'
2017-07-06 09:04:21,457 client           L1078 DEBUG| Sending command: echo $?
2017-07-06 09:04:21,558 client           L1078 DEBUG| Sending command: ethtool -i lo | awk '/bus-info/ {print $2}'
2017-07-06 09:04:21,661 client           L1078 DEBUG| Sending command: echo $?
2017-07-06 09:04:21,761 client           L1078 DEBUG| Sending command: ethtool -i rename3 | awk '/bus-info/ {print $2}'
2017-07-06 09:04:21,865 client           L1078 DEBUG| Sending command: echo $?
2017-07-06 09:04:21,966 sr_iov_hotplug   L0207 DEBUG| Interface associated with PCI ['00:01.0'] - rename3

```

File: virttest/utils_misc.py [hunk 1]
```diff
@@ -806,6 +806,64 @@ def get_full_pci_id(pci_id):
...
+    Get interface from pci id in host or in guest.
+
+    :param pci_id: PCI id of the interface to be identified
+    :param session: vm session object, if none use host interface
+    :param nic_regex: regex to match nic interfaces
+
+    :return: interface name associated with the pci id
+    """
+    if not nic_regex:
+        nic_regex = "\w+(?=: flags)|eth[0-9](?=\s*Link)"
```
Comment (luckyh) [inline virttest/utils_misc.py]: Can we use `\w+(?=\s*Link)` instead of `eth[0-9](?=\s*Link)`?

**Comment by balamuruhans** [comment]
@luckyh, Thank you. I have made changes as per your suggestion.

**Comment by balamuruhans** [comment]
@sathnaga sure, I will work on it.
@sathnaga @luckyh Thanks for the review.

**Pull request merged**